plugin/tower/pkg/controller/endpoint/controller.go

```diff
@@ -593,28 +593,43 @@ func (c *Controller) getStaticIP(key string) string {
 	return ""
 }
 
-func (c *Controller) getVMLabels(vmID string) (map[string]string, error) {
+func (c *Controller) getVMLabels(vmID string) (map[string]string, map[string][]string, error) {
 	labels, err := c.labelLister.ByIndex(vmIndex, vmID)
 	if err != nil {
-		return nil, fmt.Errorf("list labels for vm %s: %s", vmID, err)
+		return nil, nil, fmt.Errorf("list labels for vm %s: %s", vmID, err)
 	}
 
-	labelsMap := make(map[string]string, len(labels))
-	for _, label := range labels {
-		if !ValidKubernetesLabel(label.(*schema.Label)) {
-			klog.Infof("ignore vm %s valid kubernetes labels %+v", vmID, label)
+	var labelsMap = make(map[string]string)
+	var extendLabelsMap = make(map[string][]string)
+
+	for _, obj := range labels {
+		label := obj.(*schema.Label)
+		extendLabelsMap[label.Key] = append(extendLabelsMap[label.Key], label.Value)
```

Comment:
For the labels such as: [key:app]:[value:web1],[key:app]:[value:web2]
in the extendLabesMap, the key will use the [key:app], in this case will the extendLabesMap only contains: [key:app]:[value: app, web2]?

Reply:
For the above scenario, the extendLabesMap will be `[key:app]:[value: web1, web2]`.